### PR TITLE
feat(route): 中国法学网

### LINKED ISSUE #11373: 中国法学网
### 类型

学习

### 网站地址

http://iolaw.cssn.cn/

### 网站描述

中国法学网是由中国社会科学院法学研究所设立、建设和管理的法学专业网站，旨在拓展法学研究所及其员工存在和发展的虚拟空间，展示科研成果，传播法学信息，提供法律 ...

### 需要生成什么内容？

http://iolaw.cssn.cn/zxzp/

http://iolaw.cssn.cn/llwz/

等

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by HankChow**
## 该 PR 相关 Issue / Involved Issue

Close #11373 

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/cssn/iolaw/zxzp
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/en/study.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ pageClass: routes
 
 </RouteEn>
 
+## Chinese Social Science Net
+
+### Institute of Law
+
+<Route author="HankChow" example="/cssn/iolaw/zxzp" path="/cssn/iolaw/:section?" :paramsDesc="['Section ID，can be found in the URL。For example, the Section ID of URL `http://iolaw.cssn.cn/zxzp/` is `zxzp`. The default value is `zxzp`']"/>
```
Comment (TonyRL) [inline docs/en/study.md:25]:
```suggestion
<RouteEn author="HankChow" example="/cssn/iolaw/zxzp" path="/cssn/iolaw/:section?" :paramsDesc="['Section ID, can be found in the URL. For example, the Section ID of URL `http://iolaw.cssn.cn/zxzp/` is `zxzp`. The default value is `zxzp`']"/>
```

File: lib/v2/cssn/iolaw.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        .get();
+
+    ctx.state.data = {
+        title: '中国法学网',
+        url: `http://${domain}/${section}/`,
+        description: '中国法学网',
+        item: list.map((item) => ({
+            title: item.title,
+            pubDate: item.pubtime,
+            image: `http://${domain}/favicon.ico`,
```
Comment (TonyRL) [inline lib/v2/cssn/iolaw.js:34]: The property does not exist.

File: lib/v2/cssn/radar.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'cssn.cn': {
+        _name: '中国社会科学院',
+        iolaw: [
+            {
+                title: '中国法学网',
+                docs: 'https://docs.rsshub.app/study.html#cssn',
```
Comment (TonyRL) [inline lib/v2/cssn/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/study.html#zhong-guo-she-hui-ke-xue-yuan',
```

**Comment by HankChow** [comment]
updated

**Pull request merged**